api/features/multivariate/views.py

```diff
@@ -33,6 +33,21 @@ def get_permissions(self):
             )
         ]
 
+    def create(self, request, *args, **kwargs):
+        feature_pk = self.kwargs.get("feature_pk")
+
+        if feature_pk == "undefined":
+            return Response(
+                status=status.HTTP_400_BAD_REQUEST,
+                data={"detail": "Feature Id cannot be 'undefined'."},
+            )
```

Comment:
We shouldn't only check for `"undefined"` here. What about other invalid ids? Or ids of features from projects the user doesn't belong to?